src/widget/wsearchlineedit.cpp

```diff
@@ -233,7 +225,7 @@ void WSearchLineEdit::loadQueriesFromConfig() {
             m_pConfig->getKeysWithGroup(kSavedQueriesConfigGroup);
     QSet<QString> queryStrings;
     for (const auto& queryKey : queryKeys) {
-        const auto& queryString = m_pConfig->getValueString(queryKey);
+        const auto& queryString = m_pConfig->getValueString(queryKey).trimmed();
```

Comment:
Binding the result of trimmed() to a const-ref is safe, but does no longer make sense now.

Reply:
chnaged to `QString queryString = m_pConfig->getValueString(queryKey).trimmed();`
okay?